Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rust:1.86 AS build
+WORKDIR /build/vector-store-build
+RUN --mount=type=bind,target=.,rw cargo b -r && cp target/release/vector-store ..
+
+FROM ubuntu:24.04
```

Comment:
Scylla recently moved to UBI (RHEL 9 based) - see https://github.com/scylladb/scylladb/commit/1abf981a73d7c75a91ca8adfb9698a3f5e161d83

We should follow the same here.